protos/action/action.proto

```diff
@@ -74,6 +74,12 @@ service ActionService {
      * The yaw angle is in degrees (frame is NED, 0 is North, positive is clockwise).
      */
     rpc GotoLocation(GotoLocationRequest) returns(GotoLocationResponse) {}
+    /*
+     * Send command to set the value of an actuator.
+     *
+     * The index of the actuator is 1-indexed, and the value is normalized from [-1..1]
```

Comment:
I would remove this line because it is already described below.

Reply:
Ok.